ksp_plugin/celestial.hpp

```diff
@@ -39,18 +39,18 @@ class Celestial {
   void set_parent(not_null<Celestial const*> const parent);
 
   // Both accessors require |is_initialized()|.
-  Trajectory<Barycentric> const& history() const;
-  not_null<Trajectory<Barycentric>*> mutable_history();
+  Trajectory<Barycentric> const& history() const override;
+  not_null<Trajectory<Barycentric>*> mutable_history() override;
 
   // Both accessors require |is_initialized()|.
-  Trajectory<Barycentric> const& prolongation() const;
-  not_null<Trajectory<Barycentric>*> mutable_prolongation();
+  Trajectory<Barycentric> const& prolongation() const override;
+  not_null<Trajectory<Barycentric>*> mutable_prolongation() override;
 
   // Both accessors require |is_initialized()|.  In addition the first one
   // requires |has_prediction()|.
```

Comment:
Removing the contract from the interface, it doesn't make much sense there since there is no notion of `is_initialized`.